File: MaxConsecutiveOnes.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int maxStreak = 0;
        int currentStreak = 0;
        for(int i=0; i<nums.length; i++){
            if(nums[i]==1) currentStreak++;
            else{
                maxStreak = Math.max(currentStreak, maxStreak);
                currentStreak=0;
            }
        }
        return Math.max(currentStreak, maxStreak);
    }
}

File: Python/maxConsecutiveOnes.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        oneCount = 0
        maxOneCount = 0

        for num in nums:
            if num == 1:
                oneCount += 1
                maxOneCount = max(maxOneCount, oneCount)

            else:
                oneCount = 0

        return maxOneCount